src/xbps/updating.md

```diff
@@ -1,47 +1,37 @@
 # Updating
 
-Like any other system it is important to keep Void Linux up to date.
-
-In general Void should be updated with an XBPS invocation:
+Like any other system, it is important to keep Void up-to-date. Use
+[xbps-install(1)](https://man.voidlinux.org/xbps-install.1) to update:
 
 ```
 # xbps-install -Su
 ```
 
-> Note: XBPS must use a separate transaction to update itself. If your first
-> update includes the package `xbps`, you will need to run an additional update
-> for the rest of the system.
+Note: XBPS must use a separate transaction to update itself. If your update
+includes the `xbps` package, you will need to run the above command a second
+time to apply the rest of the updates.
 
-## Restarting services
+## Restarting Services
 
-If you are installing Void in production or otherwise have long lived services,
-its important to note that XBPS does not restart services when they are updated.
-This task is left to the administrator so they can orchestrate maintenance
-windows, ensure reasonable backup capacity, and generally be present for service
-upgrades.
+XBPS does not restart services when they are updated. This task is left to the
+administrator, so they can orchestrate maintenance windows, ensure reasonable
+backup capacity, and generally be present for service upgrades.
 
 To find processes running different versions than are present on disk, use the
 `xcheckrestart` tool provided by the `xtools` package:
 
-```
-# xbps-install -S xtools
-```
-
 ```
 $ xcheckrestart
 11339 /opt/google/chrome/chrome (deleted) (google-chrome)
 ```
 
 `xcheckrestart` will print out the PID, path to the executable, status of the
-path that was launched (almost always deleted) and the process name.
+path that was launched (almost always 'deleted') and the process name.
```

Comment:
single quotes or an inline code block?

Reply:
i wasn't sure about this one; i'm happy to change it to an inline code block.